templates/islandora-solr-metadata-display.tpl.php

```diff
@@ -15,7 +15,7 @@
  */
 ?>
 
-<?php if (count($solr_fields) > 1): ?>
+<?php if (count($solr_fields) > 0): ?>
```

Comment:
Something unsettling about this... Should this instead be:
```php
<?php if (count($solr_fields) > 0 || !empty($object_views_count_views)): ?>
```
... Though really... why had this been `> 1`?